### ISSUE TITLE
Parsing incomplete Date '2015 Feb'

### ISSUE THREAD

**Issue by Oli4** [dateutil/dateutil#288]
Hello everybody, I just tried to parse publication dates from NCBI. Unluckily a day is not always included and in the case of "2015 Feb" there is an ValueError. I thaugt the fallback would be the end of the month.

```
import dateutil.parser
date = dateutil.parser.parse('2015 Feb')

Traceback (most recent call last):

  File "<ipython-input-200-0ccc8304dd5e>", line 1, in <module>
    date = dateutil.parser.parse('2015 Feb')

  File "/usr/lib/python3/dist-packages/dateutil/parser.py", line 1008, in parse
    return DEFAULTPARSER.parse(timestr, **kwargs)

  File "/usr/lib/python3/dist-packages/dateutil/parser.py", line 404, in parse
    ret = default.replace(**repl)

ValueError: day is out of range for month
```

**Comment by pganssle** [comment]
@Oli4 See Issue #25, PR #30 and #191. This was fixed in version 2.5.0.

**Comment by Oli4** [comment]
@pganssle I'am sorry I did not check my version... It is working now. Thank you for the quick response.
